packages/stream_chat/lib/src/models/reaction.dart

```diff
@@ -10,13 +10,14 @@ class Reaction {
   /// Constructor used for json serialization
   Reaction({
     this.messageId,
-    required this.createdAt,
-    required this.type,
+    DateTime? createdAt,
+    this.type = '',
```

Comment:
type should be required in my oppinion

Reply:
We use it in a few places without types, although I agree in principle